hw/ip/entropy_src/doc/_index.md

```diff
@@ -10,83 +10,145 @@ This module conforms to the [Comportable guideline for peripheral functionality.
-- Two health checks that are defined by SP 800-90B are performed by this revision:  Repetition Count and Adaptive Proportion tests.
-- No alerts are supported by this revision.
-
+- Two health checks that are defined by SP 800-90B are performed by this revision: Repetition Count and Adaptive Proportion tests.
+- Two additional hardware health checks are performed by this revision: Bucket and Markov tests.
+- Firmware-defined and vendor-defined health checks are also supported.
+- A health check failure alert is supported by this revision.
 
 ## Description
 
-This IP block provides an entropy source that is part of a larger solution to generate true random numbers (a TRNG mechanism).
-The general solution for TRNG relies on either hardware or firmware to implement this function, depending which functions are enabled by firmware or e-fuses.
-For purposes of reference in this document, the NIST term "noise" will be synonymous with the term "entropy".
-Likewise, no conditioning function will be performed in this block.
-For the generation of entropy, this block provides the entropy bits needed for the entropy source solution as required by the RNG mechanism.
+This IP block provides an entropy source that is capable of using a physical noise source to generate random values in a manner that is compliant both with FIPS (though NIST SP 800-90B) and CC (AIS31) recommendations.
 
-At a high-level, this ENTROPY_SRC block, when enabled, will continuously collect entropy bits from the entropy source into a FIFO that can be read from the TLUL bus, or sent out through a hardware interface.
-The rng source will generate a relative slow set of bit streams, in the tens of kilohertz range.
-The digital source can be programmed to be faster, or much slower if desired.
-A digital source rate of flexiable speed might be useful in demonstrating how the downstream applications use the entropy.
+<!-- URLS for the following pages not upstream yet, but coming -->
+The random values generated by this block serve as non-deterministic seeds for the CSRNG block.
+The outputs of the CSRNG are then used either directly by firmware or are distributed to other hardware blocks through the Entropy Distribution Network.
+
+In the terms of AIS31 classes, this block is meant to satisfy the requirements for a PTG.2 class physical entropy source, with "internal" entropy (an AIS31 term, meaning the min-entropy as measured just *before the output pins*) exceeding 0.997 entropy-bits/output-bit.
+In NIST terms, this block satisfies the requirements for a "full-entropy" source, which requires even smaller deviations from ideal entropy, at the level of less than one part in 2<sup>64</sup>.
+When operating in full-compliance mode, raw outputs from a physical digital noise source (external to this block) are passed through a conditioning function, a secure hash function (SHA-2 or SHA-3) which has been vetted by NIST to meet these stringent requirements.
```

Comment:
double-check "physical digital noise source" is exactly correct terminology?  (as opposed to "physical random noise generator" or something else - I can never tell.)

suggestion: "passed through a conditioning function, a secure hash function (SHA-2 or SHA-3)" -> "passed through a conditioning function based on a suitable hash function (SHA-2 or SHA-3)"